servo/connectors/kubernetes.py

```diff
@@ -3324,6 +3324,8 @@ async def load_kubeconfig(self) -> None:
 DeploymentOptimization.update_forward_refs()
 CanaryOptimization.update_forward_refs()
 
+NODE_MIN_CPU = 500
+NODE_MIN_MEM = 512 * MiB
```

Comment:
If we want to land this we should prolly push this into the configuration class so that we have an escape hatch